src/borkdude/rewrite_edn/impl.cljc

```diff
@@ -29,16 +29,19 @@
                  (= :uneval (z/tag zloc)))))
           zloc))
 
-(defn indent [zloc key-count first-key-loc]
+(defn indent-or-space [zloc key-count first-key-loc]
   (let [current-loc (meta (z/node zloc))]
     (if (and first-key-loc
              (or (= 1 key-count)
                  (not= (:row first-key-loc) (:row current-loc))))
-      (let [zloc (-> zloc
-                     (z/insert-space-right (dec (dec (:col first-key-loc))))
-                     z/insert-newline-right)]
-        zloc)
-      zloc)))
+      (let [indent-spaces (dec (:col first-key-loc))]
+        (cond-> zloc
+          (> indent-spaces 0)
```

Comment:
Isn't `(> x 0)` the same as `(pos? x)`

Reply:
Yep. Agree `pos?` reads better. Will change.